plugins/services/src/js/components/HighOrderServiceConfiguration.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import Pod from "../structs/Pod";
+import PodConfigurationContainer
+  from "../containers/pod-configuration/PodConfigurationContainer";
+import Service from "../structs/Service";
+import ServiceConfigurationContainer
+  from "../containers/service-configuration/ServiceConfigurationContainer";
+
+const HighOrderServiceConfiguration = props => {
```

Comment:
It is _not_ a container since it doesn't use a store. It is a high-order component, hence I chose that name. We do this for containers as well, but please come up with some better suggestions for this, so I can use that. I do not have better suggestions myself.